public/views/block.html

```diff
@@ -59,7 +59,8 @@ <h2 translate>Summary</h2>
           </tr>
           <tr>
             <td><strong translate>Block Reward</strong></td>
-            <td class="text-right text-muted">{{$root.currency.getConvertion(block.reward) || block.reward + ' DCR'}}</td>
+            <td data-ng-show="block.height != 0" class="text-right text-muted">{{block.reward + ' DCR'}}</td>
+            <td data-ng-show="block.height == 0" class="text-right text-muted">0 DCR</td>
```

Comment:
And this one maybe:
{{$root.currency.getConvertion(0) || 0 + ' DCR'}}

assuming getConvention doesn't have some weird type requirement.